website/source/docs/platform/k8s/ambassador.html.md

```diff
@@ -0,0 +1,271 @@
+to your Kubernetes services. 
+
+
+## Installation 
+
+First, install Consul and enable Connect on the agents inside the cluster. Decide
+whether you will enable [service sync] or manually register your services with
+Consul.
+
+Once you have tested and verified that everything is working, you can proceed with
+the Ambassador installation. Full instructions are available on the [Ambassador
+site][install], but a summary of the steps is as follows:
+
+If you are deploying to GKE, create a `RoleBinding` to grant you cluster admin
+rights:
+
+```bash
+kubectl create clusterrolebinding my-cluster-admin-binding \
+        --clusterrole=cluster-admin \
+        --user=$(gcloud info --format="value(config.account)")    
+```
+
+Install Ambassador and a `LoadBalancer` service for it:
+
+```bash
+kubectl apply -f https://getambassador.io/yaml/ambassador/ambassador-rbac.yaml
+kubectl apply -f https://getambassador.io/yaml/ambassador/ambassador-service.yaml 
+```
+
+Install the Ambassador Consul Connector:
+
+```bash
+kubectl apply -f https://getambassador.io/yaml/ambassador/pro/ambassador-consul-connector.yaml
+```
+
+Add `TLSContext` and `Mapping` annotations to your existing services, directing 
+HTTPS traffic to port 20000, which is opened by the Connect proxy. Here is an 
+example of doing this for the `static-server` example used in the documentation
+for the [Connect sidecar]:
+
+```yaml
+apiVersion: v1
+kind: Service
+metadata:
+    name: static-service
+    annotations:
+    getambassador.io/config: |
+        ---
+        apiVersion: ambassador/v1
+        kind: TLSContext
+        name: ambassador-consul
+        hosts: []
+        secret: ambassador-consul-connect
+        ---
+        apiVersion: ambassador/v1
+        kind:  Mapping
+        name:  static-service_mapping
+        prefix: /echo/
+        tls: ambassador-consul
+        service: https://static-server:443
+spec:
+    type: NodePort
+    selector:
+    app: http-echo
+    ports:
+    - port: 443
+    name: https-echo
+    targetPort: 20000            
+```
+
+Once Ambassador finishes deploying, you should have a new `LoadBalancer` service
+with a public-facing IP address. Connecting to the HTTP port on this address
+should display the output from the static service.
+
+```bash
+kubectl describe service ambassador
+
+```
+
+
+## Enabling end-to-end TLS
+
+The Ambassador service definition provided in their documentation currently does not
+serve pages over HTTPS. To enable HTTPS for full end-to-end encryption, follow
+these steps.
+
+First, upload your public SSL certificate and private key as a Kubernetes secret. 
+
+```bash
+kubectl create secret tls ambassador-certs --cert=fullchain.pem --key=privkey.pem
+```
+
+Download a copy of the [ambassador-service.yaml] file from Ambassador. Replace
+the `metadata` section with one that includes an Ambassador TLS configuration block,
+using the secret name you created in step 1. Then add an entry for port 443 to the 
```

Comment:
I didn't know which `step 1` you were referring to here on the first read. I think i was looking for a `1)` rather than `First` so I didn't intuitively connect the dots. I think either adding a numerical list or just saying `previous step` would make this more clear.

Reply:
Good suggestion. Pushed 984757db2d19a82aae5809e8b7de202320f6721c to correct.